### PR TITLE
Drupal 8.9-beta2

### PR DESCRIPTION
**Pull request by alexkras**
included patch for memory leak on installation. looks like no need for future beta3.

### CODE REVIEW

**Comment by gaydabura** [comment]
@alexkras did you mean `https://git.drupalcode.org/project/drupal/-/commit/efa1e2aee7a3ca916bd1b5aad5d894.diff`

File: composer.json [hunk 1]
```diff
@@ -102,14 +110,49 @@
...
+        "Fix multilingual site's layout edit context issue": "https://www.drupal.org/files/issues/2019-12-19/3101231-19.patch",
+        "SQLite database locking errors cause fatal errors": "https://www.drupal.org/files/issues/1120020-59.patch",
+        "Error: Call to a member function getLabel() after enabling layout_builder": "https://www.drupal.org/files/issues/2019-11-14/2985882-entityfield-43.patch",
+        "Split Interface Translation permission": "https://www.drupal.org/files/issues/2020-01-14/locale-administer-locale-permission-3067056-7.patch",
+        "Memory fix leak on install": "https://git.drupalcode.org/project/drupal/-/commit/efa1e2aee7a3ca916bd1b5aad5d894,a80415a76a.diff"
+      },
+      "drush/drush": {
+        "Import multiple custom translation po files": "https://patch-diff.githubusercontent.com/raw/drush-ops/drush/pull/4251.patch"
+      },
+      "drupal/lb_ux": {
```
Comment (gaydabura) [inline composer.json:136]: i dont see this module in require section

File: composer.json [hunk 2]
```diff
@@ -14,56 +14,66 @@
...
           "type": "zip"
         }
       }
     }
   ],
   "require": {
-    "ckeditor/colorbutton": "^4.6",
-    "ckeditor/panelbutton": "^4.6",
-    "composer/installers": "^1.2",
+    "php": ">=7.0.8",
```
Comment (andypost) [inline composer.json:31]: That defined by core, do should not be here

File: composer.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "skilld-labs/drupal-project",
+  "name": "skilld-labs/sdc",
   "description": "Project template for Drupal 8 projects with composer",
   "type": "project",
-  "license": "GPL-2.0+",
+  "license": "MIT",
```
Comment (andypost) [inline composer.json:5]: Why mit?

File: composer.json [hunk 4]
```diff
@@ -14,56 +14,66 @@
...
-    "ckeditor/colorbutton": "^4.6",
-    "ckeditor/panelbutton": "^4.6",
-    "composer/installers": "^1.2",
+    "php": ">=7.0.8",
+    "ckeditor/panelbutton": "^4.10",
     "cweagans/composer-patches": "^1.6.5",
-    "drupal-composer/drupal-scaffold": "^2.5",
-    "drupal/admin_toolbar": "^1",
-    "drupal/colorbutton": "^1",
+    "drupal/better_normalizers": "^1.0@beta",
```
Comment (andypost) [inline composer.json:34]: No need for it, as we getting rid of default content

File: composer.json [hunk 5]
```diff
@@ -80,17 +90,15 @@
   },
   "scripts": {
     "drupal-scaffold": "DrupalComposer\\DrupalScaffold\\Plugin::scaffold",
+    "create-required-files": "SkilldDrupal\\composer\\ScriptHandler::createRequiredFiles",
+    "list-scaffold-files": [
+      "SkilldDrupal\\composer\\ScriptHandler::listScaffoldFiles"
+    ],
     "pre-install-cmd": [
-      "DrupalProject\\composer\\ScriptHandler::checkComposerVersion"
+      "SkilldDrupal\\composer\\ScriptHandler::checkComposerVersion"
```
Comment (andypost) [inline composer.json:98]: No need if core scaffolding enabled

Comment (andypost) [reply composer.json:98]: Also you need this patch to use core's scaffolding https://www.drupal.org/project/drupal/issues/3103090

**Comment by alexkras** [comment]
first commit was from old master, redone,
this https://www.drupal.org/project/drupal/issues/3108262 can't apply patch to 8.9.0-beta2 
will continue to check later.

File: composer.json [hunk 6]
```diff
@@ -62,7 +59,11 @@
     "drupal/webform": "^5.4",
     "drush/drush": "^10.1.0",
     "webflo/drupal-finder": "^1.1",
-    "zaporylie/composer-drupal-optimizations": "^1.0"
+    "zaporylie/composer-drupal-optimizations": "^1.0",
+    "drupal/core-composer-scaffold": "^8.8",
```
Comment (piggito) [inline composer.json:63]: Please don't add this manually, use `composer require` for that, it will keep packages sorted

File: composer.lock [hunk 7]
```diff
@@ -0,0 +1,10199 @@
+{
```
Comment (piggito) [inline composer.lock:1]: Please don't include composer.lock file

File: composer.json [hunk 8]
```diff
@@ -28,15 +28,15 @@
...
-    "php": ">=7.0.8",
     "ckeditor/panelbutton": "^4.10",
     "cweagans/composer-patches": "^1.6.5",
-    "drupal-composer/drupal-scaffold": "^2.5",
     "drupal/better_normalizers": "^1.0@beta",
     "drupal/block_content_permissions": "^1.8",
     "drupal/components": "^1",
     "drupal/config_installer": "^1",
-    "drupal/core-recommended": "^8.8.0",
+    "drupal/core-composer-scaffold": "^8.8",
```
Comment (piggito) [inline composer.json:37]: Please keep drupal-composer/drupal-scaffold we can switch to drupal/core-composer-scaffold after https://www.drupal.org/project/drupal/issues/3091285 gets fixed

File: composer.json [hunk 9]
```diff
@@ -108,6 +108,9 @@
       "web/themes/contrib/{$name}": ["type:drupal-theme"],
       "drush/contrib/{$name}": ["type:drupal-drush"]
     },
+    "drupal-core-vendor-hardening": {
+      "vendor/package": ["test", "documentation"]
```
Comment (piggito) [inline composer.json:112]: You should add real folders to clean, see https://github.com/drupal/core-vendor-hardening/blob/8.8.x/README.txt
If there aren't any you want to specify here then please remove this extra config

**Comment by piggito** [comment]
Closed in favor of https://github.com/skilld-labs/skilld-docker-container/pull/248

**Pull request closed without merging**